packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -439,7 +439,7 @@ export class RpcClient implements RpcClientInterface {
 
   async getVotingInfo(
     address: string,
-    { block }: { block: string } = defaultRPCOptions
+    { block }: { block: string; } = defaultRPCOptions
```

Comment:
were these semicolon additions done by the linter?

Reply:
either the linter or my formatting unsure which